service/tutorial/order/OrderServices.xml

```diff
@@ -0,0 +1,308 @@
+        </actions>
+    </service>
+
+    <!--    Get an Order-->
+
+    <service verb="get" noun="orderOne" displayName="get an order">
+        <in-parameters>
+            <parameter name="orderId"></parameter>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orders" type="list">
+                <parameter name="orderId"/>
+                <parameter name="orderName"/>
+                <parameter name="currencyUomId"/>
+                <parameter name="salesChannelEnumId"/>
+                <parameter name="statusId"/>
+                <parameter name="placedDate"/>
+                <parameter name="grandTotal"/>
+                <parameter name="customer_details" type="Map">
+                    <parameter name="partyId"/>
+                    <parameter name="firstName"/>
+                    <parameter name="middleName"/>
+                    <parameter name="lastName"/>
+                </parameter>
+                <parameter name="order_parts" type="list">
+                    <parameter name="orderMaps" type="Map">
+                        <parameter name="orderPartSeqId"/>
+                        <parameter name="partName"/>
+                        <parameter name="facilityId"/>
+                        <parameter name="shipmentMethodEnumId"/>
+                        <parameter name="partStatusId"/>
+                        <parameter name="partTotal"/>
+                    </parameter>
+                    <parameter name="item_details" type="list">
+                        <parameter name="orderMaps" type="Map">
+                            <parameter name="orderItemSeqId"/>
+                            <parameter name="productId"/>
+                            <parameter name="itemDescription"/>
+                            <parameter name="quantity"/>
+                            <parameter name="unitAmount"/>
+                        </parameter>
+                    </parameter>
+                </parameter>
+            </parameter>
+        </out-parameters>
+        <actions>
+            <set field="orders" from="[]"/>
+            <!-- Finding the orderId in the OrderHeader  -->
+            <entity-find entity-name="mantle.order.OrderHeader" list="orderList">
+                <econdition field-name="orderId"/>
+                <select-field
+                        field-name="orderId,orderName,currencyUomId,salesChannelEnumId,statusId,placedDate,grandTotal"/>
+            </entity-find>
+            <set field="order_parts" from="[]"/>
+            <!-- Finding the orderId in the OrderPart  -->
+            <entity-find entity-name="mantle.order.OrderPart" list="orderPartList">
+                <econdition field-name="orderId"/>
+                <select-field
+                        field-name="orderPartSeqId,partName,facilityId,shipmentMethodEnumId,statusId,partTotal,customerPartyId"/>
+            </entity-find>
+            <set field="customer_details" from="[]"/>
```

Comment:
Why is customer_details declared a list?

Reply:
we have to define a map here